substrate/utils/frame/benchmarking-cli/src/storage/mod.rs

```diff
@@ -21,3 +21,36 @@ pub mod template;
 pub mod write;
 
 pub use cmd::StorageCmd;
+
+/// Empirically, the maximum batch size for block validation should be no more than 10,000.
+/// Bigger sizes may cause problems with runtime memory allocation.
+pub(crate) const MAX_BATCH_SIZE_FOR_BLOCK_VALIDATION: usize = 10_000;
+
+pub(crate) fn get_wasm_module() -> Box<dyn sc_executor_common::wasm_runtime::WasmModule> {
+	use frame_storage_access_test_runtime::WASM_BINARY;
+	use polkadot_node_core_pvf_common::executor_interface::DEFAULT_CONFIG;
```

Comment:
We don't really need this import, because none of the settings there should influence the speed of the execution (besides simd, but we have this disabled). Also it is more clean to not import polkadot stuff here to generic FRAMe stuff (a little bit on the nitty nitpick side).